src/dashboard/cd-dashboard-stats.vue

```diff
@@ -21,10 +21,13 @@
             <div class="cd-dashboard-stats__circle-legends">
               <div v-for="gender in genderStats" class="cd-dashboard-stats__circle-legend">
                 <div class="cd-dashboard-stats__circle-legend-color" :class="[`cd-dashboard-stats__circle--${gender.name}`]"></div>
-                <div> {{ gender.name }}</div>
+                <div> {{ $t(gender.name) }}</div>
               </div>
             </div>
           </div>
+          <div v-if="femaleHintIsVisible" class="cd-dashboard-stats__chart-pie-hint">
+            <a href="https://help.coderdojo.com/hc/en-us/articles/360001143123-Increasing-the-percentage-of-girls-in-your-Dojo-">{{ $t('More informations about girls in Dojos') }}</a>
```

Comment:
I know we still have a general copy review to do, but information is a non-countable noun, so can't be pluralised.